creator-node/src/monitors/monitors.js

```diff
@@ -294,6 +294,12 @@ const MONITORS = {
   LATEST_FIND_REPLICA_SET_UPDATES_JOB_SUCCESS
 }
 
+const PROMETHEUS_MONITORS = {}
+for (const monitor in MONITORS) {
+  if (MONITORS[monitor]?.type === 'int')
```

Comment:
whats your thinking for the non-int monitors? we gonna keep writing them all to redis for now unchanged?

Reply:
yep exactly, keep as is